xAPI.md

```diff
@@ -770,16 +770,42 @@ The table below lists all properties of the Verb Object.
 	</tr>
 </table>
 
-###### Requirements
+###### Verb Id Requirements
 
-* The display property MUST be used to illustrate the meaning which is already determined by the Verb IRI.
 * A system reading a Statement MUST use the Verb IRI to infer meaning.
+* The IRI contained in an id SHOULD contain a human-readable portion which SHOULD provide meaning enough 
+to disambiguate it from other similar(in syntax) Verbs.
+* A single Verb IRI MUST NOT be used to refer to multiple meanings.
+
+###### Verb Display AP Requirements
+* The display property SHOULD be used by all Statements.
+* The display property MUST be used to illustrate the meaning which is already determined by the Verb IRI.
+
+###### Verb Display LRS Requirements
+The requirements below relate to the display property as returned by the LRS via the API.  
+
+* When queried for statements with a format of "exact", the LRS MUST return the Display property 
```

Comment:
Thanks Ben. So is that a +1 to merge as-is? 

I'll create a separate issue to expand the immutability text to incorporate verb display and mention serialisation.